lesson_10/test_homework10.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from homework_10 import log_event
+
+class MyTestCase(unittest.TestCase):
```

Comment:
Please, try to use pytest framework instead of unittest. It is better and more flexible.

Reply:
ok, i did it)
it still doesn't work :(